command/daemon/daemon.go

```diff
@@ -6,6 +6,7 @@ package daemon
 
 import (
 	"context"
+	"github.com/drone/drone-go/drone"
```

Comment:
This line should go somewhere between lines 18 and 30. Let the goimports sort them out.